scripts/sql/ai_schema_compare.py

```
@@ -60,43 +60,121 @@
-  index strategy for FKs & unique lookups, enum/check design, timestamp/timezone choices, extension usage hygiene,
-  and safe roll-forward/rollback considerations.
-- Keep each suggestion short (<= 160 chars) and actionable.
-- If a difference was ignored due to ORM constraints (e.g., DB DEFAULT present only in Flyway),
-  prefer to suggest documentation or explicit DDL in Flyway rather than flagging mismatch.
-
-{LANGUAGE_POLICY}
-""".strip()
+Explicit NON-differences for this task (CRITICAL):
+
+- Do NOT set "match": "no" just because:
+  - Schema B has STRICTER CHECK constraints (e.g. E.164 phone format, priority range 1–10),
```

Ruff (0.14.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)